Makefile

```
@@ -49,3 +49,6 @@ clean:
 	
 docs:
 	cargo doc --features test-support --no-deps --open
+
+install:
```

```suggestion
install_expand:
```
But I would remove that.

---

I would argue to leave it as is